{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2018-1000179",
      "ASSIGNER": "cve@mitre.org"
    },
    "affects": {
      "vendor": {
        "vendor_data": [

        ]
      }
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [

          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "https://github.com/quassel/quassel/blob/master/src/core/coreauthhandler.cpp#L236",
          "name": "https://github.com/quassel/quassel/blob/master/src/core/coreauthhandler.cpp#L236",
          "refsource": "CONFIRM"
        },
        {
          "url": "https://www.debian.org/security/2018/dsa-4189",
          "name": "DSA-4189",
          "refsource": "DEBIAN"
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "A NULL Pointer Dereference of CWE-476 exists in quassel version 0.12.4 in the quasselcore void CoreAuthHandler::handle(const Login &msg) coreauthhandler.cpp line 235 that allows an attacker to cause a denial of service."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [

    ]
  },
  "impact": {
  },
  "publishedDate": "2018-05-08T15:29Z",
  "lastModifiedDate": "2018-05-10T01:29Z"
}